### PR TITLE
COMMERCE-12621

### PR DESCRIPTION
**Pull request by Brunno0liveira**
https://liferay.atlassian.net/browse/COMMERCE-12621

### CODE REVIEW

**Comment by Brunno0liveira** [comment]
ci:test:relevant

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-product-test/src/testFunctional/paths/CommerceFrontStore.path [hunk 1]
```diff
@@ -180,6 +180,11 @@
 	<td>//li[contains(@class,'list-group-item')]//h5[text()='${key_entryName}']/ancestor::li//a[contains(text(),'${key_button}')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>PRODUCT_IMAGE_WITH_ALT_ATTRIBUTE</td>
+	<td>//img[contains(@alt,"thumbnail")]</td>
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/paths/CommerceFrontStore.path:185]: Please make the locator more specific, it needs to point to the mini-cart item

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductMedia.testcase [hunk 2]
```diff
@@ -257,6 +257,38 @@ definition {
...
+				productName = "U-Joint",
+				productQuantity = 1);
+
+			CommerceNavigator.gotoMiniCart();
+		}
+
+		task ("Then the admin can check the alt attribute is present") {
+			AssertElementPresent(locator1 = "CommerceFrontStore#PRODUCT_IMAGE_WITH_ALT_ATTRIBUTE");
+		}
+	}
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductMedia.testcase:290]: Assertion in the checkout process is missing.

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductMedia.testcase [hunk 3]
```diff
@@ -257,6 +257,38 @@ definition {
 		}
 	}
 
+	@description = "This is a test for COMMERCE-12621.Can view that product image cotains img tag and has the alt attribute "
+	@priority = 5
+	test CanViewAltAttributeOnImgTag {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductMedia.testcase:262]: Please use a buyer user for this test.
Also, move the test to CommerceMiniCart.testcase or CommerceCheckout.testcase

Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductMedia.testcase:262]:
```suggestion
	test CanAssertProductImagesAltAttribute {
```

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:relevant

**Comment by Brunno0liveira** [comment]
ci:stop:ci:test:stable

**Comment by Brunno0liveira** [comment]
ci:stop:stable

**Comment by Brunno0liveira** [comment]
ci:test:relevant

**Review by lorenzolapescara — APPROVED** [review]

**Comment by lorenzolapescara** [comment]
ci:test:relevant

**Comment by lorenzolapescara** [comment]
ci:test:commerce-functional

**Comment by Brunno0liveira** [comment]
ci:test:relevant

**Comment by Brunno0liveira** [comment]
ci:test:stable

**Comment by Brunno0liveira** [comment]
ci:test:stable

**Comment by victoroscuro** [comment]
ci:test:stable

**Comment by victoroscuro** [comment]
ci:test:stable

**Comment by jaydawu** [comment]
ci:test:stable

**Comment by jaydawu** [comment]
Hi @victoroscuro, the stable job is passing now, please help force-forward it, thanks!

**Comment by victoroscuro** [comment]
ci:forward:force

**Comment by victoroscuro** [comment]
ci:forward:force

**Comment by victoroscuro** [comment]
ci:forward:force

**Comment by Brunno0liveira** [comment]
PR was manually sent to Brian https://github.com/brianchandotcom/liferay-portal/pull/142997

**Comment by Brunno0liveira** [comment]
ci:close

**Pull request closed without merging**